### PR TITLE
feat(logs): enhance the logs and add a prefix with the issue number

### LINKED ISSUE #221: Prefix the logs with the issue/pr id
I think that it will help to debug if each log has a prefix with the issue or the pull request as prefix.
Also adding a separator on a row like "---" will make it less difficult to track.

### PR DESCRIPTION
**Pull request by C0ZEN**
Closes #221

### CODE REVIEW

**Comment by hross** [comment]
I like this change too except for the casing 😄 . if you want to fix that and rebase I will merge.

**Comment by hross** [comment]
I'm going to test the latest master internally for about a week and then publish a new release (and hopefully include these changes as well).

**Comment by C0ZEN** [comment]
I will handle the rebase and naming conventions tomorrow.

File: src/classes/issue-logger.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import * as core from '@actions/core';
+import {Issue} from '../IssueProcessor';
+import {Logger} from './logger';
+export class IssueLogger implements Logger {
```
Comment (C0ZEN) [inline src/classes/issue-logger.ts:4]: Add empty row to separator class and imports

**Comment by C0ZEN** [comment]
Waiting #224 merge.

**Comment by C0ZEN** [comment]
@hross and the last one is ready IMO ;)

**Review by hross — APPROVED** [review]

**Pull request merged**